test/human-data.js

```diff
@@ -141,11 +142,11 @@ describe('human-submitted app data', () => {
           expect(dimensions.width).to.equal(dimensions.height)
         })
 
-        it('is at least 128px x 128px', function () {
+        it('is at least 256px x 256px (or 128px x 128px for grandfathered apps)', function () {
           if (!fs.existsSync(iconPath)) return this.skip()
-
           const dimensions = imageSize(iconPath)
-          expect(dimensions.width).to.be.above(127)
+          const min = grandfatheredSlugs.includes(slug) ? 128 : 256
+          expect(dimensions.width).to.be.above(min - 1)
```

Comment:
I think `expect(dimensions.width).to.be.at.least(min)` will be easier for humans to understand.

Reply:
ooh didn't know about `at.least` 👍